### DISCUSSION TITLE
Progressive Web App With App Router unable to get working

### DISCUSSION THREAD

**Discussion by BradleyEvans96** [vercel/next.js#72291] [Help]
Hi,

I've followed the setup for a new next app with the following flags:
![image](https://github.com/user-attachments/assets/cc3f9e9c-c462-42c2-a61b-ecc293c6c204)

I have then followed the documentation on https://nextjs.org/docs/app/building-your-application/configuring/progressive-web-apps#next-steps to try and enable PWA within the application.

I do not seem to receive an error with the webpages and manifest however I don't seem to see the installation button within the chrome browser which is what I would expect. 

The notifications are also another ball game which haven't seemed to work with the below error being received: 
![image](https://github.com/user-attachments/assets/0aa2213c-cb86-48a6-a978-8a0ef0f2467b)

Would love some support to try and set this up correctly
Thanks in advance
Bradley

**Comment by dzaky-pr** [comment]
up

**Comment by sansan-mei** [comment]
yes, i have same isuue
